### PR TITLE
Adjusting repo for CLI (GIM-114)

### PR DESCRIPTION
**Pull request by sbilge**
- API-related files are deleted

### CODE REVIEW

File: ghga_transpiler/config.py [hunk 1]
```diff
@@ -17,17 +17,15 @@
...
-
-from .models import SupportedLanguages
+from pydantic import Extra
 
 
 # Please adapt config prefix and remove unnecessary config bases:
-@config_from_yaml(prefix="my_microservice")
-class Config(ApiConfigBase):
+@config_from_yaml(prefix="ghga_transpiler")
+class Config(ApiConfigBase, extra=Extra.allow):  # type: ignore
```
Comment (lkuchenb) [inline ghga_transpiler/config.py]: Is this needed for this PR?

File: .devcontainer/docker-compose.yml
Comment (lkuchenb) [inline .devcontainer/docker-compose.yml:1]: environment of the app service and other services should be removed

**Review by lkuchenb — CHANGES_REQUESTED** [review]

File: .devcontainer/docker-compose.yml
Comment (sbilge) [reply .devcontainer/docker-compose.yml:1]: Should I also remove ghga_transpiler_CONFIG_YAML env?

Comment (lkuchenb) [reply .devcontainer/docker-compose.yml:1]: Yes, and please also remove all config related checks (which currently make CI fail)

**Review by lkuchenb — CHANGES_REQUESTED** [review]
See above, please remove the remaining config components

**Review by lkuchenb — APPROVED** [review]

**Pull request merged**